static/js/notifications.js

```diff
@@ -262,7 +262,18 @@ function process_notification(notification) {
     // Convert the content to plain text, replacing emoji with their alt text
     content = $('<div/>').html(message.content);
     ui.replace_emoji_with_text(content);
-    content = content.text();
+
+    if (content.text().trim() === "" && content.find("p a")) {
+        var notification_message = [];
+        content.find("p a").each(function () {
+            if ($(this).attr("href").startsWith("/user_uploads/")) {
```

Comment:
We still leave the possibility of having a blank message here, perhaps. Maybe a final check like:

```js
if (notification_message.length === 0) {
    notification_message.push("Sent an attachment");
}
```